.editorconfig

```diff
@@ -2,7 +2,6 @@ root = true
 
 [*]
 indent_style = tab
-indent_size = 4
```

Comment:
Should we convert to `tab_width` instead?

Reply:
That would be fine, although is there a reason to enforce this globally at all? It seems valuable to allow users to define this locally for their own IDEs, and I wasn't finding it obvious how to tell VS Code to ignore this setting. From the [.editorconfig docs](https://editorconfig.org/#file-format-details):

> Also, when `indent_style` is set to tab, it may be desirable to leave `indent_size` unspecified so readers may view the file using their preferred indentation width.

Reply:
Oh wow, I had assumed it was entirely advisory, otherwise what's even the point of using tabs?!